### DISCUSSION TITLE
LibreScore App Installation Fix

### DISCUSSION THREAD

**Discussion by Glacitor** [LibreScore/app-librescore#43] [General]
Some help for those who may be having trouble installing the LibreScore app with LibreScore.exe.

If running the .exe file does not work, an easy fix is to extract the file with 7zip, and then run the LibreScoreInno.exe file. This will install the LibreScore app onto your computer.

Glacitor

**Comment by sealsrock12** [comment]
Or you can just [download LibreScoreInno.exe](https://github.com/LibreScore/app-librescore/releases)...
or LibreScore.zip for a portable version
